src/js/controller.js

```diff
@@ -132,7 +130,18 @@ export default class Controller extends Component {
     };
 
     triggerHashChange() {
```

Comment:
we're not triggering hashchange any more, so this method name probably ought to change

Reply:
yes will change this name @jbothma